src/PhotoSwipeGallery.js

```diff
@@ -28,9 +28,18 @@ class PhotoSwipeGallery extends React.Component {
     }
   };
 
+  thumbnails = []
   state = {
     isOpen: this.props.isOpen,
-    options: this.props.options
+    options: this.props.options || this.getThumbBoundsFn
```

Comment:
if `this.props.options` is undefined, `state.options` will be a function instead of an object?